publish/publish.proj

```diff
@@ -17,7 +17,7 @@
           Condition="'$(GitHubAuthToken)' != ''"
           DependsOnTargets="UpdatePublishedVersions" />
 
-  <Target Name="ExcludeSymbolsPackagesFromPublishedVersions" BeforeTargets="UpdateVersionsRepo" >
+  <Target Name="ExcludeSymbolsPackagesFromPublishedVersions" BeforeTargets="UpdatePublishedVersions" >
```

Comment:
Isn't this still going to

Reply:
Actually it seems like this is making the UpdateVersionsRepo target completely useless now.

Reply:
I think we need to keep `UpdateVersionsRepo` because `UpdatePublishedVersions` comes from BuildTools, and the condition doesn't make sense there. (If someone depends on `UpdatePublishedVersions` but doesn't have an auth token, an error seems like the most useful signal.)

This target will always be before `UpdateVersionsRepo` simple because `UpdateVersionsRepo` depends on `UpdatePublishedVersions`, but even if not, `UpdateVersionsRepo` doesn't use the items.

I think another way to do this would be to change `UpdateVersionsRepo` to `DependsOnTargets="ExcludeSymbolsPackagesFromPublishedVersions;UpdatePublishedVersions"` and remove the `BeforeTargets`. Maybe that would be clearer?

Reply:
yeah that would probably be a little clearer.

Reply:
Moved to DependsOnTargets and added a comment about what this stub target's doing.